hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerUnit.java

```diff
@@ -515,9 +515,9 @@ public void testLookupFileWithDnFailure() throws IOException {
         .getBlockID().getLocalID());
     assertEquals(pipelineTwo.getId(),
         newBlockLocation.getPipeline().getId());
-    assertTrue(newBlockLocation.getPipeline().getNodes().contains(dnFour));
-    assertTrue(newBlockLocation.getPipeline().getNodes().contains(dnFive));
-    assertTrue(newBlockLocation.getPipeline().getNodes().contains(dnSix));
+    assertThat(newBlockLocation.getPipeline().getNodes()).contains(dnFour);
+    assertThat(newBlockLocation.getPipeline().getNodes()).contains(dnFive);
+    assertThat(newBlockLocation.getPipeline().getNodes()).contains(dnSix);
```

Comment:
nit: we can check multiple items in a single call

```suggestion
    assertThat(newBlockLocation.getPipeline().getNodes())
        .contains(dnFour, dnFive, dnSix);
```